codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/rustlang/CargoDependency.kt

```diff
@@ -244,12 +257,11 @@ data class CargoDependency(
     companion object {
         // Forces AHash to be a later version that avoids
         // https://github.com/tkaitchuck/aHash/issues/200
-        val AHash: CargoDependency = CargoDependency("ahash", CratesIo("0.8.11"), defaultFeatures = false)
-        val OnceCell: CargoDependency = CargoDependency("once_cell", CratesIo("1.16"))
-        val Url: CargoDependency = CargoDependency("url", CratesIo("2.3.1"))
-        val Bytes: CargoDependency = CargoDependency("bytes", CratesIo("1.0.0"))
+        val AHash: CargoDependency = CargoDependency("ahash", CratesIo("0.8.11"))
```

Comment:
**question**: Was `defaultFeatures = false` being dropped from `AHash` intentional?

Reply:
This was unintentional. I've re-added this.